src-electron/db/query-session.js

```diff
@@ -492,13 +492,18 @@ async function selectSessionPartitionInfoFromDeviceType(
  * @param {*} db
  * @param {*} sessionId
  * @param {*} packageIds
+ * @param {*} onlyEnabledSessionPackages - flag to filter only enabled session packages (default is true)
  * @returns session partition info
  */
 async function selectSessionPartitionInfoFromPackageId(
   db,
   sessionId,
-  packageIds
+  packageIds,
+  onlyEnabledSessionPackages = true
```

Comment:
When it is a boolean argument it is best to use "is" prefix for better readability